### PR TITLE
Add Self-Check List Integration Test

### PR DESCRIPTION
**Pull request by Lee-JaeHyuk**
## Description
Add Self-Check List Integration Test

## Type of change
Please insert 'x' one of the type of change.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by soimkim** [comment]
@Lee-JaeHyuk ,
Please add license and copyright to the top of the file.
ex)
/*
Copyright (c) 2021 JaeHyuk Lee
SPDX-License-Identifier: AGPL-3.0-only
*/

File: src/test/java/oss/fosslight/controller/SelfCheckControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                .andExpect(status().isOk())
+                .andDo(result -> {
+                    Map<String, String> responseMap = new ObjectMapper().readValue(result.getResponse().getContentAsString(), Map.class);
+                    assertThat(responseMap.get("isValid")).isEqualTo("true");
+                });
+
+    }
+
+    @Test
+    @DisplayName("Adding Self-Check list should be failed when prjName is empty")
```
Comment (astrod) [inline src/test/java/oss/fosslight/controller/SelfCheckControllerTest.java]: Prjname of above test is also empty(""). I understand what you say. However, what do you think about putting prjName into first test(registrationShouldBeSuccess) ?

**Comment by astrod** [comment]
@Lee-JaeHyuk . The PR is same as other(https://github.com/astrod/fosslight/pull/2/files). Please reflect my comment and manage two pull request

**Review by astrod — CHANGES_REQUESTED** [review]
@Lee-JaeHyuk . The PR is same as other(https://github.com/astrod/fosslight/pull/2/files). Please reflect my comment and manage two pull request

File: src/test/java/oss/fosslight/controller/SelfCheckControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
+package oss.fosslight.controller;
```
Comment (astrod) [inline src/test/java/oss/fosslight/controller/SelfCheckControllerTest.java]: SelfCheckControllerTest and SelfCheckControllerIntegrationTest is same. Please check it, @Lee-JaeHyuk

Comment (Lee-JaeHyuk) [reply src/test/java/oss/fosslight/controller/SelfCheckControllerTest.java]: Yes, I checked. I solved it.

**Review by astrod — APPROVED** [review]
LGTM 👍

**Pull request merged**